src/Sulu/Bundle/HttpCacheBundle/DependencyInjection/SuluHttpCacheExtension.php

```diff
@@ -62,9 +62,7 @@ public function prepend(ContainerBuilder $container)
             }
         }
 
-        if (\array_key_exists('proxy_client', $fosHttpCacheConfig)) {
-            $fosHttpCacheConfig['tags']['enabled'] = $config['tags']['enabled'];
-        }
+        $fosHttpCacheConfig['tags']['enabled'] = $config['tags']['enabled'];
```

Comment:
@vntw at the end it makes sense to have that - but the tests failing ... seems also to be checked in the foshttpcachebundle